src/Microsoft.Identity.Client/Instance/AdfsOpenIdConfigurationEndpointManager.cs

```diff
@@ -169,37 +133,16 @@ protected override void AddToValidatedAuthorities(string userPrincipalName)
                        string.Format(
                            CultureInfo.InvariantCulture,
                            "https://enterpriseregistration.windows.net/{0}/enrollmentserver/contract",
```

Comment:
>"https://enterpriseregistration.windows.net/{0}/enrollmentserver/contract", [](start = 27, length = 75)

I prefer stuff like this to be in consts (class level or specialed classes for e.g. ADFS or OpenId or...)